### ISSUE TITLE
Can retention be exposed as a cmd line parameter?

### ISSUE THREAD

**Issue by jshook** [tureus/graphite-rust#1]
https://github.com/tureus/graphite-rust/blob/master/src/bin/carbon.rs#L59-L64

**Comment by xrl** [comment]
@jshook yes, I think that would be an easy change. Put the property in the `Args` struct here: https://github.com/tureus/graphite-rust/blob/master/src/bin/carbon.rs#L35-L40 and document the argument in the `USAGE` string (I think the formatting matters): https://github.com/tureus/graphite-rust/blob/master/src/bin/carbon.rs#L19-L32 . Do that and I will be happy to merge!
